recipe/meta.yaml

```diff
@@ -30,29 +30,30 @@ requirements:
     - pip
   run:
     - python
-    - asteval >=0.9.12
+    - asteval >=0.9.18
     - fabio
-    - h5py >=2.2
-    - lmfit >=0.9.9
-    - matplotlib >=1.5.1
+    - h5py >=2.8
+    - lmfit >=1.0
+    - matplotlib >=3.0
     - numpy >=1.7
     - peakutils >=1.0.0
     - pillow >=3.4
     - psutil
     - pyFAI
+    - pyyaml
     - requests >=2.1
     - scikit-image
     - scikit-learn
-    - scipy >=0.17
+    - scipy >=1.2
     - silx
     - six >=1.10
     - sqlalchemy >=0.9
     - termcolor
-    - tomopy
     - uncertainties
     - wxmplot
     - wxpython
     - wxutils
+    - xraydb >=4.3
```

Comment:
@dmgav, where did these pins/new packages come from? I'll probably need to port them to #4.